libraries/chain/include/eosio/chain/block_header_state.hpp

```diff
@@ -41,7 +41,12 @@ struct level_2_commitments_t {
 struct finality_digest_data_v1 {
    uint32_t    major_version{light_header_protocol_version_major};
    uint32_t    minor_version{light_header_protocol_version_minor};
-   uint32_t    active_finalizer_policy_generation {0};
+   uint32_t    active_finalizer_policy_generation{0};
+
+   // Hold the generation number of pending_finalizer_policy if it exists,
+   // otherwise hold the same value as active_finalizer_policy_generation
+   uint32_t    pending_finalizer_policy_generation{0};
```

Comment:
If called `last_pending_finalizer_policy_generation` we can just have a short comment after the semicolon.